lib/probe/definitions.ex

```diff
@@ -78,21 +78,23 @@ defmodule Instruments.Probe.Definitions do
           [name]
       end
 
-    GenServer.call(__MODULE__, {:define, definitions, defn_fn})
+    unique_names = unique_names(probe_names, options)
+
+    GenServer.call(__MODULE__, {:define, probe_names, unique_names, defn_fn})
   end
 
-  def handle_call({:define, probe_names, transaction}, _from, _) do
+  @spec handle_call({:define, [String.t()], [String.t()], (() -> any)}, any, any) ::
+          {:ok, [String.t()]} | {:error, {:probe_names_taken, [String.t()]}}
+  def handle_call({:define, probe_names, unique_names, transaction}, _from, _) do
```

Comment:
looks good, but i don't think we need `probe_names`? appears this is only used in the return value, but it seems reasonable to me to to return `unique_names` here? or just even nothing.

Reply:
sounds good, done